config/jobs/kubernetes/sig-autoscaling/azure/cluster-autoscaler-azure-presubmits.yaml

```diff
@@ -21,7 +21,7 @@ presubmits:
     extra_refs:
     - org: kubernetes-sigs
       repo: cluster-api-provider-azure
-      base_ref: release-1.16
+      base_ref: release-1.18
```

Comment:
Are we planning to update the rest of the jobs here like this once we see this one seem stable?

Reply:
ya I'll create a follow up once we confirm these changes on the master job